src/test/kotlin/com/bardsoftware/papeeria/backend/cosmas/CosmasGoogleCloudServiceTest.kt

```diff
@@ -116,6 +117,39 @@ class CosmasGoogleCloudServiceTest {
         assertEquals("kek lol", file)
     }
 
+    @Test
+    fun restoreToBufferCreatePatch() {
+        val fakeStorage: Storage = mock(Storage::class.java)
+        val blob1 = getMockedBlob("ver1", 0)
+        val blob2 = getMockedBlob("ver2", 1)
+        Mockito.`when`(fakeStorage.get(eq(service.getBlobId(FILE_ID, projectInfo(), null))))
+                .thenReturn(blob1).thenReturn(blob2)
```

Comment:
I suggest moving the second `thenReturn` lower after call to `addPatchToService` to make it clear that we return blob2 only after that

Reply:
ok